yellowbrick/features/importances.py

```diff
@@ -257,7 +295,8 @@ def _is_fitted(self):
 ##########################################################################
 
 def feature_importances(model, X, y=None, ax=None, labels=None,
-                        relative=True, absolute=False, xlabel=None, **kwargs):
+                        relative=True, absolute=False, xlabel=None,
+                        stack=False, **kwargs):
```

Comment:
Just a reminder if you wouldn't mind modifying the docstring to include the `stack` attribute; I'd appreciate it!

Reply:
Yeap~ I will add it to docstring.

Reply:
Yeap~ I will add it to docstring.